x-pack/plugin/core/src/main/java/org/elasticsearch/license/LicensedFeature.java

```diff
@@ -88,6 +88,22 @@ protected LicensedFeature(String family, String name, License.OperationMode mini
         this.needsActive = needsActive;
     }
 
+    public String getFamily() {
```

Comment:
Can you please change the declarations above to be private now that there are accessors?